ui/.env

```diff
@@ -0,0 +1,8 @@
+#####
+# UI ENVIRONMENT VARIABLEs
+# Copy and configure the variables below in your own `.env.local` or `.env.[mode].local` file
+# https://vitejs.dev/guide/env-and-mode.html#env-files
+#####
+
+# Redux logger ('true' to enable)
+VITE_REDUX_LOGGER_ENABLED=
```

Comment:
Would this mean that its undefined by default or what does leaving out the right hand side of the `=` result in ?

Reply:
It ends up as an empty string. Environment variables are just strings, so in the code, I test for the string `"true"`.